pkg/machine/machine_common.go

```diff
@@ -131,7 +131,7 @@ address can't be used by podman. `
 					fmtString += `If you would like to install it, run the following commands:
 
         sudo %s install
-        podman machine stop%[1]s; podman machine start%[1]s
+        podman machine stop%[2]s; podman machine start%[2]s
```

Comment:
there are a few other `[1]s` instances above this line within this function.  Do they need tending to also?

Reply:
probably yes, but i need to verify that

Reply:
yes, fixed